test/integration/cert_options_test.go

```diff
@@ -80,3 +80,32 @@ func TestCertOptions(t *testing.T) {
 	}
 
 }
+
+// TestCertExpiration makes sure minikube can start after its profile certs have expired.
+// It does this by configuring minikube certs to expire after 3 minutes, then waiting 3 minutes, then starting again.
+// It also makes sure minikube prints a cert expiration warning to the user.
+func TestCertExpiration(t *testing.T) {
+	MaybeParallel(t)
```

Comment:
will this affect parallel tests that use the same CA ? or would it use its own minikube home ?

Reply:
--cert-expiration doesn't change anything about the CA, just the profile specific certs